examples/language_modeling-tf.ipynb

```diff
@@ -6,7 +6,7 @@
     "id": "X4cRE8IbIrIV"
```

Comment:
<p>There is nothing to uncomment anymore here. I'd leave out the login part as we don't want people to rely on it anymore, it soon won't be possible anymore.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/137/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>